tests/tests_snowflake/test_snowflake_base.py

```diff
@@ -3,6 +3,7 @@
 from llama_hub.snowflake.base import SnowflakeReader
 from llama_index.readers.schema.base import Document
 
+
```

Comment:
also this? or was this a result from lint/fmt?

Reply:
^ yep, this was also from the linter - i can remove from this PR if you prefer.

Reply:
all good, let's just leave them both thanks!